scripts/test-install-zip.sh

```diff
@@ -56,15 +71,20 @@ rm -r build/packages/*/
 
 # Install packages and verify
 for zipFile in build/packages/*.zip; do
-  PACKAGE_NAME_VERSION=$(basename ${zipFile} .zip)
+  installAndVerifyPackage ${zipFile}
+done
 
-  elastic-package install -v --zip ${zipFile}
+# try to install one package without elastic-package stack shellinit
+unset ELASTIC_PACKAGE_KIBANA_HOST
+unset ELASTIC_PACKAGE_ELASTICSEARCH_USERNAME
+unset ELASTIC_PACKAGE_ELASTICSEARCH_PASSWORD
+unset ELASTIC_PACKAGE_CA_CERT
```

Comment:
You can remove the shellinit eval instead of doing these `unset`s, right?

Reply:
If I remove the eval, the previous loop would not work. Another way would be directly exporting the new values.